composables/use-log-event.ts

```diff
@@ -23,3 +26,71 @@ export default function useLogEvent(eventName: string, eventParams?: EventParams
     }
   }
 }
+
+export function useLogSetUser(user: {
+  email?: string
+  evmWallet?: string
+  likeWallet?: string
+  loginMethod?: string
+  displayName?: string
+  avatar?: string
+  id?: string
+} | null) {
```

Comment:
Is the type equal to: https://github.com/likecoin/3ook-com/blob/c7897fda2b44fcdb20e18a7f6d018d62fd86f3b5/types/nuxt-auth-utils.d.ts#L2-L15